tracker/common/dcp_agents/analysis_agent.py

```diff
@@ -2,37 +2,43 @@
 import os
 import json
 
-from cromwell_tools import api as cwm_api
-from cromwell_tools import cromwell_auth as cwm_auth
+import cromwell_tools
+from typing import List, Dict
 
 
 class AnalysisAgent:
-    def __init__(self):
+    def __init__(self) -> None:
+        """Interacts with the HCA DCP Pipelines Execution Service."""
         deployment = os.environ['DEPLOYMENT_STAGE']
         self.cromwell_url = 'https://cromwell.caas-prod.broadinstitute.org'
         self.cromwell_collection = 'lira-test' if deployment == 'integration' else f'lira-{deployment}'
         gcp_credentials_file_for_analysis = os.environ.get('GCP_ACCOUNT_ANALYSIS_INFO')
-        if gcp_credentials_file_for_analysis:
-            with open(gcp_credentials_file_for_analysis, 'r') as f:
-                service_account_key = json.load(f)
-            self.auth = self._get_auth(service_account_key)
+        self.auth = cromwell_tools.cromwell_auth.CromwellAuth.harmonize_credentials(service_account_key=gcp_credentials_file_for_analysis,
```

Comment:
Will need to merge in master. This is updated a bit as the secret is now stored in aws secrets manager

Reply:
done!